nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/web/StandardNiFiConnectorWebContext.java

```diff
@@ -28,13 +28,13 @@ public class StandardNiFiConnectorWebContext implements NiFiConnectorWebContext
     private ConnectorDAO connectorDAO;
 
     @Override
-    @SuppressWarnings("unchecked")
-    public <T> T getConnector(final String connectorId) throws IllegalArgumentException {
+    @SuppressWarnings({ "unchecked"})
+    public <T> ConnectorWebContext<T> getConnectorWebContext(String connectorId) throws IllegalArgumentException {
```

Comment:
```suggestion
    public <T> ConnectorWebContext<T> getConnectorWebContext(final String connectorId) throws IllegalArgumentException {
```
Method parameters should be marked `final`.